static/src/javascripts/projects/commercial/modules/messenger/scroll.ts

```diff
@@ -122,16 +141,18 @@ const removeScrollListener = (iframe) => {
 	}
 };
 
-const onMessage = (respond, start, iframe) => {
+const isCallable = (x: unknown): x is Respond => typeof x === 'function';
+
+const onMessage: Parameters<RegisterListener>[1] = (respond, start, iframe) => {
```

Comment:
Would it be worth making a note to export / import from commercial-core `ListenerCallback` directly?

Reply:
decided to just inline it like the rest of the messenger modules, was exported in `_` but never used in the test.